docs/source/conf.py

```diff
@@ -284,6 +284,8 @@ def document_plugins_tokens_list(list_path: Path, output_path: Path) -> None:
         ),
         "version_match": os.environ.get("READTHEDOCS_VERSION", "latest"),
     },
+    # rely on browser-native accessibility features instead of custom JS
+    "navigation_with_keys": False,
```

Comment:
Removing as it is gonna end up twice in the configuration with the fix from #15310 

```suggestion
```